.alexrc

```diff
@@ -1,3 +1,3 @@
 {
-    "allow": ["invalid", "color", "colors", "white", "latin", "hook", "hooks", "trap"]
+    "allow": ["invalid", "color", "colors", "white", "latin", "hook", "hooks", "trap", "whitespace", "just", "special"]
```

Comment:
**note:** This covers the warnings that were failing alex. The alternative is to rephrase the CHANGELOGs that were causing the issues.